<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工大动态</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/images.qrc">
    <normaloff>:/HITers.ico</normaloff>:/HITers.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section{background:rgb(85, 170, 255)};</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>标题</string>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>浏览量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonNews">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工大新闻</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNotice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通知公告</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonScience">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学术科研</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res/images.qrc"/>
 </resources>
 <connections/>
</ui>
